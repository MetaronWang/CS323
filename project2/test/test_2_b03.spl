struct a{
    int c;
    int d;
};
struct b{
    int c;
    int d;
};
struct a1{
    struct a aa;
    int i;
};

struct b1{
    struct b bb;
    int i;
};

struct a2{
    struct b1 bb;
    int i;
};

int main(){
    struct a1 aa;
    struct b1 bb;
    struct a2 aaa;
    aa = bb;
    aa = aaa;
}