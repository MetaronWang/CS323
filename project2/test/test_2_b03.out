Error type 5 at Line 29: unmatching types on both sides of assignment operator (=)
